vendorconsent/tcf2/metadata.go

```diff
@@ -76,6 +82,7 @@ func (c ConsentMetadata) Created() time.Time {
 	return time.Unix(deciseconds/decisPerOne, (deciseconds%decisPerOne)*nanosPerDeci)
 }
 
+// LastUpdated returns the last updated date stored in bits 42 to 77
```

Comment:
43 to 78.. Ok. I'll stop here. You get the idea. :)

Please update the bit ranges in the comments by 1 or somehow else make it clear that we're counting from 0. Appreciate adding all of the comments though.

Reply:
ok